aptos-move/framework/src/natives/cryptography/bulletproofs.rs

```diff
@@ -34,11 +34,19 @@ pub mod abort_codes {
 
     /// Abort code when input value for a range proof is too large.
     /// NOTE: This must match the code in the Move implementation
-    pub const NFE_VALUE_OUTSIDE_RANGE: u64 = 0x01_0002;
```

Comment:
Again, these shouldn't be changed. I think it will lead to backwards compatibility errors.